src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "acl_eltwise.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace arm_compute;
+
+TensorShape eltwiseShapeCast(const VectorDims &dims) {
+    arm_compute::TensorShape tensorShape;
+    for (std::size_t i = 0; i < dims.size(); ++i) {
+        tensorShape.set(dims.size() - i - 1, dims[i], false);
+    }
+    if (tensorShape.num_dimensions() == 0) {
+        tensorShape.set(0, 1, false);
+        tensorShape.set_num_dimensions(1);
+    }
+    return tensorShape;
+}
+
+AclEltwiseExecutor::AclEltwiseExecutor() : EltwiseExecutor() {}
+
+bool AclEltwiseExecutor::init(const EltwiseAttrs &eltwiseAttrs, const std::vector<MemoryDescPtr> &srcDescs,
+                              const std::vector<MemoryDescPtr> &dstDescs,
+                              const std::vector<EltwisePostOp> &postOps) {
+    aclEltwiseAttrs = eltwiseAttrs;
+
+    for (const auto &desc : srcDescs) {
+        if (desc->getShape().getRank() > 4)
```

Comment:
Same comment as for MVN. There are greate amount of operations which shapes is not equal to 4D. We need to use the same approach as for MVN: lets merge some dimension into one in case rank>4 or add explicit 1s into shape in case rank<4 assuming broadcasting semantics.